source/main/audio/SoundScriptManager.h

```diff
@@ -254,82 +254,87 @@ class SoundScriptInstance : public ZeroedMemoryAllocator
 
     // functions
-    void trigOnce    (int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void trigOnce    (Actor* actor, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void trigStart   (int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void trigStart   (Actor* actor, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void trigStop    (int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void trigStop    (Actor* actor, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void trigToggle  (int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void trigToggle  (Actor* actor, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void trigKill	 (int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
-    void trigKill    (Actor* actor, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
-    bool getTrigState(int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    bool getTrigState(Actor* actor, int trig, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void modulate    (int actor_id, int mod, float value, int linkType = SL_DEFAULT, int linkItemID=-1);
-    void modulate    (Actor* actor, int mod, float value, int linkType = SL_DEFAULT, int linkItemID=-1);
+    void trigOnce(int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigOnce(Actor *actor, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigStart(int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigStart(Actor *actor, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigStop(int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigStop(Actor *actor, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigToggle(int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigToggle(Actor *actor, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigKill(int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void trigKill(Actor *actor, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    bool getTrigState(int actor_id, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    bool getTrigState(Actor *actor, int trig, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void modulate(int actor_id, int mod, float value, int linkType = SL_DEFAULT, int linkItemID = -1);
+    void modulate(Actor *actor, int mod, float value, int linkType = SL_DEFAULT, int linkItemID = -1);
 
     void setEnabled(bool state);
 
     void setCamera(Ogre::Vector3 position, Ogre::Vector3 direction, Ogre::Vector3 up, Ogre::Vector3 velocity);
-    void setLoadingBaseSounds(bool value) { loading_base = value; };

```

Comment:
I'm quite fond of these short inline getters/setters, but it seems ClangFormat has no support for it - there's just BraceWrapping.AfterFunction.